recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: d2cd8a1de1c498c1832f0765113ad58086d1d3b29bf5d2cc9b1ba60f1919951c
 
 build:
-    number: 0
+    number: 1
```

Comment:
There is no need to bump because we did not change anything from build `0`, we just added the build `0` for `OS X`. (But you do not need to change it either. It is a minor thing.)